prow/plugins/yuks/yuks.go

```diff
@@ -101,6 +101,20 @@ func handleGenericComment(pc plugins.Agent, e github.GenericCommentEvent) error
 	return handle(pc.GitHubClient, pc.Logger, &e, jokeURL)
 }
 
+func escapeMarkdown(s string) string {
```

Comment:
Could we get a unit test that shows the input/output for this and highlights edges?

Reply:
Added some unit tests. Even through in a markdown joke, just for you @stevekuznetsov 😉